bindings/java/src/junit/com/apple/foundationdb/tuple/ArrayUtilTests.java

```diff
@@ -20,79 +20,77 @@
 
 package com.apple.foundationdb.tuple;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
 
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
-import java.util.Random;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.Ignore;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
 /**
  * @author Ben
  *
  */
-public class ArrayUtilTests {
+ class ArrayUtilTests {
 
 	/**
 	 * Test method for {@link ByteArrayUtil#join(byte[], java.util.List)}.
 	 */
 	@Test
-	public void testJoinByteArrayListOfbyte() {
-		byte[] a = new byte[] {'a', 'b', 'c'};
-		byte[] b = new byte[] {'d', 'e', 'f'};
+	 void testJoinByteArrayListOfbyte() {
```

Comment:
whitespace.

`s/public//g` <- what I think you did
`s/public //g` <- what I think you meant